### DISCUSSION TITLE
Adding arrows to start and end of the line

### DISCUSSION THREAD

**Discussion by stockiNail** [chartjs/chartjs-plugin-annotation#545] [Ideas]
I'd like to propose to add arrows at the beginning and ending of a line annotation.

Options:

```javascript
{
  type: 'line',
  ....
  arrows: { 
    enabled: boolean,
    start: {
      type: 'none' | 'full' | 'empty' | 'point' | 'rect' and others....
      borderWidth: number
      borderColor: color,
      radius: number,
      others...
    },
    end: {
      type: 'none' | 'full' | 'empty' | 'point' | 'rect' and others....
      borderWidth: number
      borderColor: color,
      radius: number
      others...
    }   
  }
}
```

start and end can be set as boolean to activate them with defaults.

What do you think?

**Comment by kurkle** [comment]
Could be useful. I think we should be getting 1.2.0 out though and take some feedback before adding more enhancements.

**Reply by stockiNail** [reply]
yes, absolutely. I was just thinking loud. I think version 2 or later. There are other enhancements (for instance text decoration, as it's implemented in Chart,js, if I'm not wrong)

**Comment by stockiNail** [comment]
Arrow heads are available from version [1.3.0](https://github.com/chartjs/chartjs-plugin-annotation/releases/tag/v1.3.0)
